closed/Intel/code/resnet50/pytorch-cpu/run_offline_accuracy.sh

```diff
@@ -30,30 +30,30 @@ if [ -z "${RN50_FULL}" ]; then
     exit 1
 fi
 
-CONDA_ENV_NAME=rn50-mlperf
-source ~/anaconda3/etc/profile.d/conda.sh
-conda activate ${CONDA_ENV_NAME}
+#CONDA_ENV_NAME=rn50-mlperf
+#source ~/anaconda3/etc/profile.d/conda.sh
+#conda activate ${CONDA_ENV_NAME}
 
 export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
 
-export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
+#export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
 
-export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
+#export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
 
 KMP_SETTING="KMP_AFFINITY=granularity=fine,compact,1,0"
 export KMP_BLOCKTIME=1
 export $KMP_SETTING
 
 CUR_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-APP=${CUR_DIR}/build/bin/mlperf_runner
+APP=${PWD}/build/bin/mlperf_runner
 
 export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CONDA_PREFIX}/lib
 
 if [ -e mlperf_log_accuracy.json ]; then
     rm mlperf_log_accuracy.json
 fi
 
-numactl -C 0-55,56-111 -m 0,1 ${APP} --scenario Offline \
+numactl -C 0-47 -m 0 ${APP} --scenario Offline \
```

Comment:
one quick question, why do we have a fix number 47 here?

Reply:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/113460727/200607144-c9df971e-d060-4780-baf2-c606edd18080.png)
I met this issue saying that "55 and 56-111" is invalid. After checking lscpu, I change the number to "0-47"